src/main/webapp/app/lti/manage/lti-course-card/lti-course-card.component.spec.ts

```diff
@@ -3,21 +3,32 @@ import { LtiCourseCardComponent } from 'app/lti/manage/lti-course-card/lti-cours
 import { ARTEMIS_DEFAULT_COLOR } from 'app/app.constants';
 import { ActivatedRoute } from '@angular/router';
 import { of } from 'rxjs';
+import { Directive, Input } from '@angular/core';
+import { TranslateService } from '@ngx-translate/core';
+import { MockTranslateService } from 'test/helpers/mocks/service/mock-translate.service';
+import dayjs from 'dayjs/esm';
 
+@Directive({
+    selector: '[jhiTranslate]',
+})
+class MockTranslateDirective {
+    @Input() jhiTranslate: string;
+}
 describe('LtiCourseCardComponent', () => {
     let component: LtiCourseCardComponent;
     let fixture: ComponentFixture<LtiCourseCardComponent>;
 
     beforeEach(async () => {
         await TestBed.configureTestingModule({
-            imports: [LtiCourseCardComponent],
+            imports: [LtiCourseCardComponent, MockTranslateDirective],
```

Comment:
```suggestion
            imports: [LtiCourseCardComponent, MockDirective(TranslateDirective)],
```

Please use this way to mock the translate directive. Then you can get rid of your custom MockTranslateDirective :)

Reply:
Worked, thx :D